From 8bbbaaaf78ecd479d149fc84ab7d50be90c3540d Mon Sep 17 00:00:00 2001
From: Peter Xu <peterx@redhat.com>
Date: Thu, 17 Mar 2016 09:19:27 +0800
Subject: [PATCH 063/134] trace-cmd: Keep default bash completion to list files

This will keep trace-cmd default completion behave like before.

Signed-off-by: Peter Xu <peterx@redhat.com>
Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 trace-cmd.bash | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/trace-cmd.bash b/trace-cmd.bash
index 3b76af2d6750..c7c5cae629ef 100644
--- a/trace-cmd.bash
+++ b/trace-cmd.bash
@@ -42,8 +42,9 @@ _trace_cmd_complete()
             COMPREPLY=( $(compgen -W "${funcs}" -- "${cur}") )
             ;;
         *)
-            # By default, we do not provide any hints.
-            COMPREPLY=()
+            # By default, we list files
+            local files=$(ls --color=never)
+            COMPREPLY=( $(compgen -W "${files}" -- "$cur") )
             ;;
     esac
 }
-- 
2.5.5

